docs/source/guides/auth.mdx

```diff
@@ -62,17 +62,18 @@ This guide uses [Auth0](https://auth0.com/) as the Identity Provider.
 
 The Auth0 Connection is the method clients use to authenticate. This guide uses the default **Username-Password-Authentication** connection.
 
-1. In your dashboard, navigate to **Authentication** -> **Database**.
-1. Create the default **Username-Password-Authentication** connection. Click the **Try Connection** button to test it.
-1. Copy the **Connection Identifier** at the top of the page, you need it for the next step. It should start with something like `con_`.
+1. In your Auth0 dashboard, navigate to **Authentication** -> **Database**.
+1. Create the default **Username-Password-Authentication** connection. Click the **Try Connection** button to test it and set up a username and password for later.
+1. Back on your Auth0 dashboard, copy the **Connection Identifier** at the top of the page. You'll need it soon. It should start with something like `con_`.
 1. Navigate to **Applications** -> **APIs** -> **Auth0 Management API**.
-1. Click the **API Explorer** tab. Copy the token value, you also need it for the next step.
-1. Run the following `curl` command to promote the connection to domain-level, replacing `CONNECTION_ID` and `MGMT_API_ACCESS_TOKEN` with the values you copied in the previous steps:
+1. Copy the **Identifier** for this API. It should look something like `dev-123456.us.auth0.com`, where `dev-123456` is your Auth0 tenant ID.
```

Comment:
@jcaudle Is "Identifier" the correct word here? It should be Auth0 domain, right?

Reply:
OK so it looks like that's called the Auth0 Tenant ID https://auth0.com/docs/get-started/tenant-settings/find-your-tenant-name-or-tenant-id

I made reference to it in Step 2. Can we update to use that instead?